### ISSUE TITLE
Discord is logged out after site refresh or browser restart

### ISSUE THREAD

**Issue by Alpengreis** [uBlockOrigin/uBlock-issues#1993]
### Prerequisites

- [X] I verified that this is not a filter issue (MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues))
- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### Description

Discord is logged out after site refresh or browser restart. This is pretty new behaviour, maybe since uBlock Dev. 1.41.5b1 or 1.41.5b2 ... That behaviour begins just after install of uBlock Dev 1.41.5b2 with default settings.

It remains after disabling or even deinstalling of uBlock extension then! It only good again after reset the Firefox-Settings or after creating a new Firefox profile.

Please note that I had to check the option "The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser" above, because else I could not post the issue. BUT: as I sais it's even present after disabling uBblock!

### A specific URL where the issue occurs

discord.com

### Steps to Reproduce

1. Login to discord
2. Refresh discord site or restart browser

### Expected behavior

Should be still logged in

### Actual behavior

It's logged out

### uBlock Origin version

1.41.5b2

### Browser name and version

Firefox Developer 98.0b6

### Operating System and version

Microsoft Windows 10 Pro x64 [19044.1566] [21H2] [de-CH]

**Comment by Alpengreis** [comment]
Hmm ... currently, I can't reproduce it ... needs more testing. Close it for now ...

**Comment by gorhill** [comment]
Please next time you have such issues which you want to immediately attribute to uBO without actual hard evidence, do not open an issue on the tracker, either open a new thread in Discussions or use Reddit for support.

**Comment by Alpengreis** [comment]
Sure, sorry about that. The thing was: I was sure about it FIRST ...

Just for clarification: I found out what is responsible for that behaviour - at least I am almost sure about ;-)

It's the case after I activated the bookmark panel (<Ctrl>+B). So it seems it's a browser problem and has nothing to do with uBlock. That was REALLY difficult to find out FOR ME! But I thought, I write it here yet, just for the case, if other users have the same problem ...

Once again, sorry for the trouble - I know you have more than enough things to check here always!
